src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/GlobalRunSpec.cs

```diff
@@ -111,6 +149,51 @@ private static IReadOnlyList<IOperationProvider> SetupOperations(IComponentManag
             return operations;
         }
 
+        private static IReadOnlyList<IOperationProvider> SetupCustomOperations(IReadOnlyList<ICustomOperationModel> customModel, IComponentManager componentManager, IDirectory templateRoot, IVariableCollection variables, IParameterSet parameters)
+        {
+            List<IOperationProvider> customOperations = new List<IOperationProvider>();
+
+            foreach (ICustomOperationModel opModelUntyped in customModel)
+            {
+                CustomOperationModel opModel = opModelUntyped as CustomOperationModel;
+                if (opModel == null)
+                {
+                    continue;   // TODO: decide if this is ok, or if we need some other handling.
```

Comment:
if the host interface could be accessible here - this would be a good case to issue a warning back to the broker for